Build/libHttpClient.GDK.props

```diff
@@ -51,9 +61,17 @@
   <!-- Include the main MS props -->
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
 
-  <PropertyGroup>
+  <!-- When using plain x64 platform name, set paths manually -->
+  <PropertyGroup Condition="'$(Platform)'=='x64'">
+    <GDKLibPath Condition="'$(GDKLibPath)'==''">$(GameDKLatest)windows\lib\x64</GDKLibPath>
```

Comment:
Keep in mind this directory reference only works with GDK 2510 or later.

Reply:
Yeah, I think this is fine